src/crimson/mon/MonClient.cc

```diff
@@ -541,7 +541,7 @@ Client::ms_dispatch(crimson::net::Connection* conn, MessageRef m)
   }
 }
 
-seastar::future<> Client::ms_handle_reset(crimson::net::ConnectionRef conn, bool is_replace)
+seastar::future<> Client::_ms_handle_reset(crimson::net::ConnectionRef conn, bool is_replace)
```

Comment:
is the '_' a formal requirement now in our code?

Reply:
Oh, I added '_' just because the original methods are rewritten to call the ones with '_'.

Reply:
not a formal requirement. i double checked https://google.github.io/styleguide/cppguide.html and https://github.com/ceph/ceph/blob/master/CodingStyle, but failed to find anything supporting this.

but personally, i found this could reduce the mental overhead to finding a new name =) normally, we also add `do_` before the helper private function which does the heavy lift. but i think the "_" also works.